src/Task_1/DoWhile.java

```diff
@@ -0,0 +1,51 @@
+package Task_1;
```

Comment:
Let's stick to convention in regards to spacing:
https://www.oracle.com/technetwork/java/javase/documentation/codeconventions-141388.html#:~:text=One%20blank%20line%20should%20always%20be%20used%20in%20the%20following%20circumstances%3A&text=1)%20or%20single%2Dline%20(,a%20method%20to%20improve%20readability

Reply:
Also, you can hit Ctrl+Shift+L while working on your code to format it.
Also, you can hit the 'Format code' flag in the commit window if you use IntelliJ

Reply:
Thanks, will study this rule of spacing/empty lines.